### PR TITLE
#13 Issue Resolved : android.app.ProgressDialog is deprecated as of API 26 

### PR DESCRIPTION
**Pull request by jatinjain29**
>ProgressDialog is deprecated, so Progress Bar is added instead of it
>Since Progress Bar does not have any function like setmessage() to display a message, so TextView is used instead to show the message that was shown before using setMessage() function.
>A linear layout is added that contains both the text view and progress bar to collectively serve as a Progress Dialog

### CODE REVIEW

File: app/src/main/res/layout/activity_login.xml [hunk 1]
```diff
@@ -23,6 +23,33 @@
...
+            android:layout_height="match_parent"
+            android:layout_weight="1" />
+
+        <TextView
+            android:layout_width="0dp"
+            android:layout_height="match_parent"
+            android:layout_weight="4"
+            android:gravity="center"
+            android:id="@+id/tv_progressMsg"
+            android:text="Hey! By using this app you can find Great workspaces." />
```
Comment (riturajjain2000) [inline app/src/main/res/layout/activity_login.xml:50]: Change this message to `loading`

**Review by riturajjain2000 — CHANGES_REQUESTED** [review]
Resolve merge Conflicts

**Comment by jatinjain29** [comment]
Merge conflicts are resolved and the message is changed to Loading.

**Review by riturajjain2000 — APPROVED** [review]

**Comment by jatinjain29** [comment]
Someone else has also changed the same code so that was creating merge conflict, I have resolved it. Can you please confirm?

**Comment by riturajjain2000** [comment]
> Someone else has also changed the same code so that was creating merge conflict, I have resolved it. Can you please confirm?

yes it is now good . Can you confirm that the app is still running?

**Comment by jatinjain29** [comment]
> > Someone else has also changed the same code so that was creating merge conflict, I have resolved it. Can you please confirm?
> 
> yes it is now good . Can you confirm that the app is still running and you did not break anything else?

>yes it is running well

**Review by riturajjain2000 — APPROVED** [review]

**Pull request merged**